### PR TITLE
Add watch for data files to gulpfile

### PR DESCRIPTION
**Pull request by kball**
Datafiles changing should also auto-trigger a rebuild

### CODE REVIEW

**Comment by marvinhuebner** [comment]
Any possibilities to get this merged?

File: gulpfile.babel.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ function watch() {
   gulp.watch(PATHS.assets, copy);
   gulp.watch('src/pages/**/*.html').on('all', gulp.series(pages, browser.reload));
   gulp.watch('src/{layouts,partials}/**/*.html').on('all', gulp.series(resetPages, pages, browser.reload));
+  gulp.watch('src/data/**/*').on('all', gulp.series(resetPages, pages, browser.reload));
```
Comment (marvinhuebner) [inline gulpfile.babel.js:138]: Panini does only process `.js, .json, .yml` files as i can see: https://github.com/zurb/panini/blob/master/lib/loadData.js#L11

So we could only watch these file extensions. 

Could look like: 
```js
gulp.watch('src/data/**/*.{js,json,yml}').on('all', gulp.series(resetPages, pages, browser.reload));
```

**Review by marvinhuebner — COMMENTED** [review]
Comment is only a little improvement. But it is possible to get this PR merged?

File: gulpfile.babel.js (hunk 1 above)
Comment (ncoden) [reply gulpfile.babel.js:138]: I @marvinhuebner 👋. Sorry for the late reply. You are right, I'll close this PR in favor of yours.

**Comment by ncoden** [comment]
Closing in favor of https://github.com/zurb/foundation-zurb-template/pull/63.

**Pull request closed without merging**